src/panels/lovelace/editor/card-editor/hui-dialog-delete-card.ts

```diff
@@ -45,10 +45,11 @@ export class HuiDialogDeleteCard extends LitElement {
           ${this._cardConfig
             ? html`
                 <div class="element-preview">
-                  <hui-card-preview
+                  <hui-card
                     .hass=${this.hass}
                     .config=${this._cardConfig}
-                  ></hui-card-preview>
+                    .editMode=${true}
```

Comment:
```suggestion
                    editMode
```

Reply:
edit mode isn't a attribute for now but I will convert it.